bespin/settings.py

```diff
@@ -32,6 +33,10 @@
         # 'ATOMIC_REQUESTS': True,
     }
 }
+# Update database configuration from $DATABASE_URL if set
+db_from_env = dj_database_url.config()
```

Comment:
Nice find on the `dj-database-url` library. I think we should use it for dev and production too for consistency, not just CI. And the more I think about it, the less I want to encourage using sqlite here. I'll create an issue for that, and we can update the ansible role too.

This file could use some documentation on how to construct such a database URL, since it's not a django built-in project. That can just be an example URL and link to the README though.

Reply:
Created #124